engine/class_modules/sc_death_knight.cpp

```diff
@@ -4635,16 +4639,6 @@ struct deaths_due_t : public death_and_decay_base_t
 
     death_and_decay_base_t::execute();
   }
-
-  bool ready() override
-  {
-    if ( p() -> talent.defile -> ok() )
-    {
-      return false;
-    }
-
-    return death_and_decay_base_t::ready();
-  }
 };
```

Comment:
What's the plan with defile talented? This implementation would make both cast of Defile and death's due available (athough any_dnd handles it, that will not work for specific deaths_due or defile action names).